spec/controllers/presentations_controller_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe PresentationsController, type: :controller do
+  let(:admin_person) { FactoryGirl.build(:person, :admin) }
+
+  before do
+    allow_any_instance_of(PresentationsController).to receive(:pundit_user)
```

Comment:
Something about how this is written doesn't make me feel confident that we're testing this code properly. Do we need to stub this?